streamer/src/nonblocking/quic.rs

```diff
@@ -132,6 +134,31 @@ fn prune_unstaked_connection_table(
     }
 }
 
+fn get_connection_stake(connection: &Connection, staked_nodes: Arc<RwLock<StakedNodes>>) -> u64 {
+    connection
+        .peer_identity()
+        .and_then(|der_cert_any| der_cert_any.downcast::<Vec<rustls::Certificate>>().ok())
+        .and_then(|der_certs| {
+            der_certs.first().and_then(|der_cert| {
+                X509Certificate::from_der(der_cert.as_ref())
+                    .ok()
+                    .and_then(|(_, cert)| {
+                        cert.public_key().parsed().ok().and_then(|key| match key {
+                            PublicKey::Unknown(inner_key) => {
+                                let pubkey = Pubkey::new(inner_key);
+                                info!("Peer public key is {:?}", pubkey);
```

Comment:
Probably shouldn't be `info!` level. This would be printed on every connection, right?